gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
```

Comment:
Lurer på om du også må kjøre en eller annen kommando for å generere nye gradle-wrapper-filer og sjekke inn disse. Se tilsvarende PR i ispersonoppgave https://github.com/navikt/ispersonoppgave/pull/209/files

Reply:
https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper

Reply:
Aha. Jeg får vel kanskje noen feilmeldinger hvis jeg tester i dev slik som den er nå

Reply:
Bra sett! 🤩 Tror det skal være greit nå

Reply:
Ser ut som `gradlew.bat` filen allerede har de endringene som ble gjort i PR'en i `ispersonoppgave`